client/src/domains/constants.ts

```diff
@@ -7,6 +7,18 @@ export const MOBILE_MAX_WIDTH = 575.98;
 
 export const PAGE_CNT = 5;
 
+export const MENU_HEIGHT = 50;
+export const TUTORIAL_CONTENTS = {
+  EDITOR: "1. 에디터고 여기서 편집을 할 수 있으며 기본으로 제공되는 양식은 발표의 첫 페이지이며 없애면 이상해져요.",
+  PREVIEW: "2. 마크다운 적용 결과고요 미리 볼 수 있어요.",
```

Comment:
You can check the rendered content entered in the editor on the preview screen in real time.
Keep in mind that the pages are divided with a thick line(\<hr>).